### PR TITLE
profiles: two step flow screens & reverse records

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes RNBW-2609
Fixes RNBW-2753
Fixes RNBW-2787

## What changed (plus any additional context for devs)

Added different components for each of the registration flows, we just need the animations now so I didn't put effort on the UI and added a simple loader
Added ability to send or not a reverse record tx, it will default to true only if the user doesn't have a reverse record already
Updated registration cost panel to latest design

## PoW (screenshots / screen recordings)

https://recordit.co/dE2EYtj9Ou

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels?

### CODE REVIEW

**Review by pugson — APPROVED** [review]

**Review by benisgold — APPROVED** [review]
have some problems building on android rn but looks good on ios!

File: src/screens/ENSConfirmRegisterSheet.js [hunk 1]
```diff
@@ -2,17 +2,21 @@ import { useFocusEffect, useRoute } from '@react-navigation/core';
...
 import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { Switch } from 'react-native-gesture-handler';
 import { useRecoilState } from 'recoil';
 import brain from '../assets/brain.png';
 import ActivityIndicator from '../components/ActivityIndicator';
 import Spinner from '../components/Spinner';
 import { HoldToAuthorizeButton } from '../components/buttons';
 import { RegistrationReviewRows } from '../components/ens-registration';
 import { GasSpeedButton } from '../components/gas';
+import { Centered } from '../components/layout';
```
Comment (jxom) [inline src/screens/ENSConfirmRegisterSheet.js:13]: prefer design-system?

Comment (benisgold) [reply src/screens/ENSConfirmRegisterSheet.js:13]: blasphemy

**Review by jxom — APPROVED** [review]

File: src/screens/ENSConfirmRegisterSheet.js (hunk 1 above)
Comment (estebanmino) [reply src/screens/ENSConfirmRegisterSheet.js:13]: :O

Comment (estebanmino) [reply src/screens/ENSConfirmRegisterSheet.js]: ok done!

**Pull request merged**